.travis.yml

```diff
@@ -28,3 +39,9 @@ script:
 
 after_success:
   - coveralls
+  - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
```

Comment:
Actually is is something I want to improve, build a image every time and run tests on top of it.